File: 06webWiew/app/src/main/java/com/example/a06webwiew/MainActivity.java
package com.example.a06webwiew;

import android.os.Bundle;
import android.webkit.WebView; // Importa la clase WebView correctamente

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    private WebView webView; // Corrige el nombre de la variable

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        webView = findViewById(R.id.webView); // Corrige el nombre de la variable
        webView.loadUrl("https://www.cebem.net/index.php/es/");
    }
}

File: 044recyclerview-en-android/Diferentes_Layouts/src/main/java/com/develou/diferentes_layouts/MainActivity.kt
package com.develou.diferentes_layouts

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.RecyclerView
import com.develou.common.Article

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.articles_view_types)

        val articleList: RecyclerView = findViewById(R.id.list)

        val articleAdapter = ArticleAdapter { article -> onItemClick(article) }
        articleList.adapter = articleAdapter

        val list = listOf(DataItem.HeaderItem) + Article.data.map { DataItem.ArticleItem(it) }
        articleAdapter.submitList(list)
    }

    private fun onItemClick(article: Article) {
        Toast.makeText(this, article.title, Toast.LENGTH_SHORT).show()
    }
}